content/applications/sales/subscriptions/closing.rst

```diff
@@ -1,105 +1,115 @@
+button at the top of the :guilabel:`In Progress` subscription order. This option is also available
+after the order has been invoiced and the payment has been registered.
 
-.. image:: closing/close-your-subscriptions-as-an-administrator.png
+.. image:: closing/close-subscriptions-administrator.png
   :align: center
-  :alt: Close your subscription from an administration point of view with Odoo Subscriptions
+  :alt: Close subscription from an administration point of view with Odoo Subscriptions.
 
-By using the smart button *Close*, you have to enter a close reason. For example, "Subscription too
-expensive", "Subscription does not meet my requirements", "Subscription reached its end date", etc.
-Immediately after confirming your close reason, you can observe that the status of the subscription
-is now *Closed* and that the close reason is mentioned on the subscription.
+Clicking the :guilabel:`Close` button prompts a :guilabel:`Close Reason` pop-up window to appear,
+allowing administrators to input the reason for closing the subscription, or choose from the
+drop-down menu of options in the :guilabel:`Reason` field.
 
-.. image:: closing/use-of-close-reasons.png
+.. image:: closing/close-reason-popup.png
   :align: center
-  :alt: What happens when you close your subscription with Odoo Subscriptions?
+  :alt: The Close Reason pop-up that appears when the Close button is clicked in Odoo Subscriptions.
 
-Customer view
--------------
+When the desired :guilabel:`Reason` is entered, click the :guilabel:`Submit` button.
+
+Clicking :guilabel:`Submit` on the :guilabel:`Close Reason` pop-up window updates the subscription
+sales order to show :guilabel:`Churned` status tag, along with the specified :guilabel:`Close
+Reason`.
+
+.. image:: closing/churned-sales-order.png
+  :align: center
+  :alt: A churned sales order for a closed subscription in Odoo Subscriptions.
 
-As previously explained in the *Administrator view* part, from the subscription form, you also have
-the possibility to visualize what your customers see when managing their subscriptions thanks to the
-*Customer preview* button. In this example, the customer has the choice to close his/her subscription
-whenever he/she wants to, due to the *Close Subscription* button.
+That same close reason can be found in the *Chatter* of the sales order, as well.
 
-.. image:: closing/close-your-subscriptions-as-a-customer.png
+.. image:: closing/churned-sales-order-chatter.png
   :align: center
-  :alt: Close your subscription from a customer point of view with Odoo Subscriptions
+  :alt: The chatter of a churned sales order for a closed subscription in Odoo Subscriptions.
+
+Customer view
+-------------
+
+.. note::
+  Administrators can use the :guilabel:`Preview` button at the top pf the subscriptions sales order
+  to view the the customer's perspective, where the customer can find the :guilabel:`Close
+  Subscription` button on the left side of the sales order in the customer portal.
```

Comment:
The text in an admonition need to be lined up with the first character in the admonition type title, `n` in `.. note::`:
```suggestion
.. note::
   As an administrator, the ability to visualize what customers see when managing their
   subscriptions is accessible via the :guilabel:`Preview` button, located at the top of the
   subscription sales order.
```